welcome.css

```diff
@@ -72,6 +71,82 @@ html, body {
 
 .text-tag {
   text-align: center;
-  font-size: 32px;
+  font-size: 38px;
   color: olive;
+}
+
+.burger-icon:before {
+  display: none;
+  font-size: 32px;
+}
+
+.click-icon {
+  display: none;
+}
+
+@media (max-width: 800px) {
+  .burger-icon:before{
+    display: block;
+    margin-top: 3%;
+  }
+
+  .menu-btn {
+    position: absolute;
+    right: 0;
+    margin: 8px;
+    z-index: 3;
+  }
+
+  .click-icon:checked ~ .menu-btn i:before{
+    content: "\f00d";
+  }
+
+  .click-icon:checked ~ .menu-btn {
+    color: white;
+    font-size: 32px;
+  }
+
+  .menu-list {
+   position: fixed;
+   width: 100%;
+   display: none;
+   text-align: center;
+   transition: all 0.3s ease;
+   opacity: 90%;
+  }
+
+  .web-name-container {
+    width: 100%;
+  } 
+
+  .click-icon:checked ~ .menu-list{
```

Comment:
Samely, please tell me why you write this code.

Reply:
".click-icon:checked ~ .menu-list{ "  means that if someone click "click-icon" button open the menu.

<img width="659" alt="Screenshot 2024-07-10 at 1 22 37 PM" src="https://github.com/scheme-dojo/scheme-dojo.github.io/assets/143992531/eec278d3-6aa1-42e1-9d00-8a89c520a0e5">

Reply:
Just modify the name` .click-icon`.
What role does this icon play?